display_list/dl_builder.cc

```diff
@@ -1786,6 +1812,63 @@ bool DisplayListBuilder::AccumulateBounds(const SkRect& bounds,
   return true;
 }
 
+bool DisplayListBuilder::LayerInfo::update_clear_color(DlColor color,
+                                                       DlBlendMode mode) {
+  if (still_clearing) {
+    switch (mode) {
```

Comment:
We should be able to support all blend modes, if we clear more than 1 color we perform the blending on the CPU. clear color should not impact  max_blend_mode I think